src/components/ItemList/ItemList.module.css

```diff
@@ -1,12 +1,60 @@
 .wrap {
+  min-height: 150px;
+  max-height: 400px;
   padding-bottom: 15px;
+  padding-left: 0;
+  overflow-y: scroll;
+}
+
+.wrap .checkbox__icon {
+  color: #356eff;
+}
+
+.wrap .delete__icon {
+  color: #ccc;
+}
+
+.wrap .checkbox {
+  margin-left: 0;
 }
 
 .item__wrap {
   display: flex;
   justify-content: space-between;
+  width: 634px;
+  height: 36px;
+  border: 1px solid #CCCCCC;
+  border-radius: 41px;
 }
 
 li {
   list-style-type: none;
+  padding-bottom: 8px;
+}
+
+.checkbox {
```

Comment:
Codacy found an issue: [Expected selector ".checkbox" to come before selector ".wrap .checkbox" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5830965)